dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -135,10 +135,8 @@ export default defineComponent({
           );
           if (axiosResponse.status == 200) {
             this.isUserDataOwner = true;
-            console.log(axiosResponse);
           }
         } catch (error) {
-          console.error(error);
           this.isUserDataOwner = false;
           if (getErrorMessage(error).includes("404")) {
             this.isUserDataOwner = false;
```

Comment:
Ich denke diese drei Zeilen machen so keinen Sinn.

Wenn man doch sowieso immer `this.isUserDataOwner = false` setzt, was bringt dann die if-Schleife noch?

Reply:
Hast du recht, sind entfernt